.github/workflows/rspec.yml

```
@@ -0,0 +1,14 @@
+name: RSpec
+on: [push, pull_request]
+jobs:
+  rspec:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: '2.3.1'
+        bundler-cache: true
+    - run: gem install bundler -v '< 2.0'
+    - run: bundle install
+    - run: bundle exec rake spec
```

nit: I'd like to see add indent.

```suggestion
      - uses: actions/checkout@v3
      - uses: ruby/setup-ruby@v1
        with:
          ruby-version: '2.3.1'
          bundler-cache: true
      - run: gem install bundler -v '< 2.0'
      - run: bundle install
      - run: bundle exec rake spec
```